src/modules/general/components/SearchDropdown/index.tsx

```diff
@@ -85,7 +86,7 @@ export const SearchDropdown: React.FC<SelectProps> = ({
   ...props
 }) => {
   const selectedVal = props.value || '';
-  const selectRef = useRef<HTMLSelectElement>(null);
+  const selectRef = useRef<any>(null);
```

Comment:
instead use `Select<any>` and Select is from 'react-select`

Reply:
It doesn't resolves ts error. Can you try it?